sn_interface/src/network_knowledge/section_authority_provider.rs

```diff
@@ -325,5 +350,18 @@ pub mod test_utils {
 
             (signed_sap, nodes, secret_key_set.secret_key())
         }
+
+        pub fn _gen_sk_set<R: RngCore>(
```

Comment:
How about this instead?
```suggestion
        pub fn gen_sk_set_with_threshold<R: RngCore>(
```